l10n_ch_pain_base/models/account_payment_order.py

```diff
@@ -90,7 +90,7 @@ def generate_party_acc_number(
             if not partner_bank.ccp:
                 raise UserError(_(
                     "The field 'CCP/CP-Konto' is not set on the bank "
-                    "account '%s'.") % partner_bank.name)
+                    "account '%s'.") % partner_bank.bank_name)
```

Comment:
You don't want bank_name here, you want acc_number to be relevant. If you had multiple account on the same bank you wouldn't know where the error message came from.

This should have been addressed already in https://github.com/OCA/l10n-switzerland/pull/425

I guess the error in multiple odoo versions.

I'll backport the fix.